pkg/run/stack.go

```diff
@@ -167,9 +168,16 @@ func (r *RunStackState) WebsocketsTable(port int) (string, int) {
 func (r *RunStackState) TopicTable() (string, int) {
 	tableData := pterm.TableData{{"Topic", "Endpoint"}}
 
-	for k, address := range r.subs {
+	topicKeys := make([]string, 0, len(r.subs))
+	for k := range r.subs {
+		topicKeys = append(topicKeys, k)
+	}
```

Comment:
can probably use https://github.com/samber/lo#keys for this.

Reply:
Done